src/data/enums/theme.enums.ts

```diff
@@ -0,0 +1,4 @@
+export enum UserThemes {
+  DARK = 'dark',
+  LIGHT = 'light',
+}
```

Comment:
Let's name it just Themes.

```suggestion
export enum UserThemes {
  DARK = 'dark',
  LIGHT = 'light',
}
```